charts/otel-cloud-stack/values.yaml

```diff
@@ -39,6 +39,8 @@ autoinstrumentation:
 
 daemonCollector:
   name: daemon
+  labels:
```

Comment:
i want to only apply these if the bridge is enabled....

Reply:
maybe have a variable here that's like `enable_opamp_management` and then conditionally add this label in `bridge.yaml` template?

Reply:
i wonder if it makes sense to add this in collector.yaml (similar line of thinking):
```
labels:
{{- if $Values.opampBridge.enabled }}
  opentelemetry.io/opamp-managed: "true"
{{- end }}
```